monero-outreach-docs/translations/es/quickfacts/Quick-facts_es_printerfriendly.sla

```diff
@@ -718,14 +593,14 @@
+        <PAGEOBJECT OwnPage="5" PTYPE="4" XPOS="391" YPOS="4399" WIDTH="240" HEIGHT="110" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFEATURES="inherit underline" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="17" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="MonereoSerif Bold Italic" ISIZE="14" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 240 0 240 0 240 0 240 0 240 110 240 110 240 110 240 110 0 110 0 110 0 110 0 110 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 240 0 240 0 240 0 240 0 240 110 240 110 240 110 240 110 0 110 0 110 0 110 0 110 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" ALIGN="2" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
             <ITEXT FONT="MonereoSerif Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="Evitar estafas (scams)"/>
             <para ALIGN="2" LINESP="17"/>
             <ITEXT FONT="MonereoSerif Bold Italic" CH="Preguntas Frecuentes"/>
             <para LINESP="17"/>
             <ITEXT FONT="MonereoSerif Bold Italic" CH="Análisis SWOT de Monero"/>
             <para LINESP="17"/>
-            <ITEXT FONT="MonereoSerif Bold Italic" CH="Conectarse con la Comunidad de Monero"/>
+            <ITEXT FONT="MonereoSerif Bold Italic" CH="Conectarse con la Comunidad deMonero"/>
```

Comment:
Why change this? The space was right.

Reply:
Think it was an error of mine. Some sentences were separated not sure why. Maybe taking spaces and organizing the text was when I did this. Let me fix it.